new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1155 @@
+
+The substitution command is followed by a pattern to search for, then the
+replacement and finally a series of option flags. There are other flages as
+well, but 'g' is the most important one. The pattern is a regular expression,
+but in this example we cared only about a specific word so we used the literal
+string we wanted.
+
+Here are a few more examples of what is possible with the substitute command:
+
+  4. To change every occurrence of a character string between two lines, type
+~~~ viml
+     :#,#s/old/new/g
+~~~
+     where #,# are the line numbers of the range of lines where the
+     substitution is to be done.
+
+     Type
+~~~ viml
+     :%s/old/new/g
+~~~
+     to change every occurrence in the whole file.
+
+     Type
+~~~ viml
+     :%s/old/new/gc
+~~~
+     to find every occurrence in the whole file, with a prompt whether to
+     substitute or not.
+
+# Lesson 2.8: THE SET OPTION COMMAND
+
+** Setting Vim options is done using a command **
+
+  1. Type ':set list' to make Vim display whitespace characters.
+
+  2. Type ':set nolist' to make Vim hide whitespace characters.
+
+  3. Type ':set list!' to toggle the option on and off.
+
+  4. Type ':set invlist' to toggle the option as well.
+
+  5. Type ':set list?' to print the current state of the option.
+
+Most options in Vim are boolean values. The syntax for set set command is
+~~~ viml
+     :set [no]option
+~~~
+where 'option' is the name of the option. The option name without prefix always
+turns it on and with the 'no' prefix always turns it off. The exclamation mark
+postfix or the 'inv' prefix always toggles between on and off.
+
+Some actions take values instead:
+
+  5. Type ':set foldcolumn=4' to make Vim display a column to the left
+
+You can also reset any option to its default:
+
+  6. Type ':set foldcolumn&' to reset.
+
+There are more ways to manipulate options, see the help manual for all of them.
+Check the manual for every individual option as well, usually there is an
+abbreviated form of them as well.
+
+# Lesson 2.9: SEARCH MODE
+
+** Type «/» followed by a pattern to search for the pattern. **
+
+This is not really a command but it fits here. Press the «/» key and type the
+word "fox". The cursor will jump to the word on the line below.
+
+---> The brown fox jumps over the lazy dog.
+
+We can repeat this search using the «n» (next) operator. You will cycle through
+all the matches, so keep pressing it until you are back here. To search in the
+other direction use «N».
+
+We can also issue a search in the opposite direction using the «?» key. Search
+again for the word "fox". Try it out.
+
+As you have seen the «n» and «N» keys are now reversed as well. It's easier to
+issue one backwards search and then keep hitting «n» than hitting «N»
+repeatedly.
+
+The pattern does not have to be just a word, it can be any regular expression.
+Try searching for "f.x". You will find matches for both "fox" and "fix".
+
+
+# Summary
+
+asdf
```

Comment:
perhaps this should be dealt with before merging :grin:

I think it's fine to just remove it for now.